doc/cephfs/best-practices.rst

```diff
@@ -30,13 +30,13 @@ and **do not use snapshots**.  Both of these are the default.
 
 Note that creating multiple MDS daemons is fine, as these will simply be
 used as standbys.  However, for best stability you should avoid
-adjusting ``max_mds`` upwards, as this would cause multiple
+adjusting ``max_mds`` upwards, as this would cause multiple MDS
 daemons to be active at once.
 
 Which client?
 -------------
 
-The fuse client is the easiest way to get up to date code, while
+The fuse client is easy to setup, while
```

Comment:
-1, it's correct to say that the fuse client is the easiest way to get feature parity with the MDSs.

Reply:
@batrick I have changed it, because I found it's difficult for the user to understand what `get up to date code` exactly means. Do you have a better sentence ? Or I will just revert my change.

Reply:
How about: "The FUSE client is the most accessible and the easiest to upgrade to the version of Ceph used by the storage cluster, while"

Reply:
@batrick 

> The fuse client is the easiest way to get up to date code

I think this sentence originated from the fact that using the kernel client (stable release) might give older features than the fuse client, as it might be a little bit older.

So what about this sentence ?
`The FUSE client is the most accessible and the easiest way to get the latest cephfs features, while the kernel client will often give better performance.`

Reply:
"latest cephfs features" is dependent on whether the Ceph cluster is as new as the client.

Reply:
@batrick Fixed. Thanks.